### PR TITLE
Actually use double quote in quoteIfSpaced according to the docs. #649

### PR DESCRIPTION
**Pull request by ptek**

### CODE REVIEW

**Comment by ptek** [comment]
Issue reference: #649

**Comment by simonmichael** [comment]
https://github.com/simonmichael/hledger/issues/649#issuecomment-374239013

File: hledger-lib/Hledger/Utils/Text.hs [hunk 1]
```diff
@@ -123,7 +124,11 @@ textElideRight width t =
 quoteIfSpaced :: T.Text -> T.Text
 quoteIfSpaced s | isSingleQuoted s || isDoubleQuoted s = s
                 | not $ any (`elem` (T.unpack s)) whitespacechars = s
-                | otherwise = "'"<>escapeSingleQuotes s<>"'"
+<<<<<<< HEAD
+                | otherwise = "\""<>escapeSingleQuotes s<>"\""
+=======
```
Comment (mparker17) [inline hledger-lib/Hledger/Utils/Text.hs]: It looks like a merge conflict accidentally got committed here

**Comment by ptek** [comment]
Oh, I am sorry, I did not realize that my pushes actually ended up here
Will fix it now

**Comment by ptek** [comment]
This should fix it. I am only uncertain how to write unit tests

**Pull request merged**